### DISCUSSION TITLE
Gatsby icon and image resources.. ok in localhost BUT do not show when 'public' gatsby build

### DISCUSSION THREAD

**Discussion by johnda98** [gatsbyjs/gatsby#30665] [Help]
<!--Gatsby icon and image resources.. ok in localhost BUT do not show when 'public' gatsby build pinned to IPFS as static site.

gatsby develop.. all just dandy fine

gatsby clean cool..

gatsby build.. cool

localhost imga resources appear... no when pinned to IPFS via IPFS desktop.. the hashlink to the gatsby build output 'public' folder works well. Except that the image resources do not show or in one case, blurred.

any help appreciated.. as not many gatsby sites on ipfs yet i think

**Comment by johnda98** [comment]


**Reply by johnda98** [reply]
could be it https://www.gatsbyjs.com/plugins/gatsby-plugin-ipfs/   hang on ..

pathPrefix: '__GATSBY_IPFS_PATH_PREFIX__',

and  start_url: '__GATSBY_IPFS_PATH_PREFIX__',

**Comment by LekoArts** [comment]
You'll need to provide some more information if you want good responses ;)

Have you tried using/reading these?

https://github.com/moxystudio/gatsby-plugin-ipfs
https://github.com/christroutner/gatsby-ipfs-boilerplate
https://medium.com/cladular/building-deploying-a-gatsby-website-to-ipfs-using-azure-pipelines-7dd095a861fb
https://blog.fleek.co/posts/Gatsby-Fleek

**Reply by johnda98** [reply]
thanks Leko.. resolved :-))   and build with path prefix.. did the trick :-)
